packages/core/src/components/Tooltip/styles.ts

```diff
@@ -55,7 +55,10 @@ export const styleSheetTooltip: StyleSheet = ({ unit, color, pattern, ui }) => (
   },
 
   content_inverted: {
-    backgroundColor: color.accent.blackout,
+    backgroundColor:
+      mode === 'light'
+        ? '#181818' // Dark mode base color
+        : '#FFF', // Light mode base color
```

Comment:
do these exist on `color`?

Reply:
Only on the opposite theme 🤔 . Should be solved by using the new `baseInverse`